src/jobs/jobs.service.ts

```
@@ -3,11 +3,19 @@ import {
-  Inject,
 } from '@nestjs/common';
-import { InjectRepository } from '@nestjs/typeorm';
-import { Repository, DataSource, Brackets } from 'typeorm';
-import { CompletionStatus, Job, JobStatus } from './entities/job.entity';
+import { InjectRepository, InjectDataSource } from '@nestjs/typeorm';
+import {
+  Repository,
+  DataSource,
+  Brackets,
+  LessThan,
+  MoreThanOrEqual,
```

ESLint

[error] 'MoreThanOrEqual' is defined but never used.

(@typescript-eslint/no-unused-vars)